rootfs/scheduler/__init__.py

```diff
@@ -622,23 +622,22 @@ def _create_rc(self, name, image, command, **kwargs):
         containers[0]['args'] = args
         loc = locals().copy()
         loc.update(re.match(MATCH, container_fullname).groupdict())
-        mem = kwargs.get('memory', {}).get(loc['c_type'])
-        cpu = kwargs.get('cpu', {}).get(loc['c_type'])
+        mem = kwargs.get('memory', {})
+        cpu = kwargs.get('cpu', {})
         env = kwargs.get('envs', {})
         if env:
             for k, v in env.iteritems():
                 containers[0]["env"].append({"name": k, "value": v})
         if mem or cpu:
             containers[0]["resources"] = {"limits": {}}
-
```

Comment:
would you mind keeping these formatting changes out of this PR? I'm sure @helgi has a few words to say about these changes ;)